lib/src/boom_menu.dart

```diff
@@ -216,18 +227,24 @@ class _BoomMenuState extends State<BoomMenu> with SingleTickerProviderStateMixin
     );
 
     return Positioned(
-      left: widget.marginLeft - 16,
-      bottom: widget.marginBottom - 16,
-      right: widget.marginRight - 16,
       child: Container(
         child: Column(
           mainAxisAlignment: MainAxisAlignment.end,
           crossAxisAlignment: CrossAxisAlignment.end,
           children: List.from(fabChildren)
             ..add(
-              Container(
-                margin: EdgeInsets.only(top: 8.0, right: 2.0),
-                child: animatedFloatingButton,
+              Align(
+                alignment: widget.fabAlignment,
+                child: Container(
+                  padding: EdgeInsets.only(
+                    left: 28.0 + widget.fabPaddingLeft,
+                    right: widget.fabPaddingRight,
+                    top: widget.fabPaddingTop,
```

Comment:
top: widget.fabPaddingTop,
Remove this Line(92). it's make a Bottom overflowed error.